Pipfile

```diff
@@ -10,8 +10,8 @@ python-dotenv = "*"
 requests = "*"
 aserto-idp = "*"
 typing-extensions = "*"
-flask-aserto = "~=0.20.0"
-aserto-directory = "*"
+flask-aserto = "~=0.20.1"
+aserto = "*"
```

Comment:
I think you may want to put `>= 0.21` to ensure you get a version that has the directory client.